docs/site/src/pages/reference/python-client/stream-data.mdx

```diff
@@ -52,6 +90,17 @@ for frame in streamer:
 
 <Divider.Divider direction="x" />
 
+## Updating the Channel List
+
+If you want to update the list of channels being streamed, you can call the
+`update_channels` method on the streamer:
```

Comment:
Can you be clear that `update_channels` replaces the list of channels that are currently being streamed?